framework/src/modules/dpos_v2/module.ts

```diff
@@ -395,13 +394,26 @@ export class DPoSModule extends BaseModule {
 
 		const initDelegates = [...genesisStore.genesisData.initDelegates];
 		initDelegates.sort((a, b) => a.compare(b));
-		const bftWeights = initDelegates.map(d => ({
-			bftWeight: BigInt(1),
-			address: d,
-		}));
 		const initBFTThreshold = BigInt(Math.floor((2 * initDelegates.length) / 3) + 1);
-		await this._bftAPI.setBFTParameters(apiContext, initBFTThreshold, initBFTThreshold, bftWeights);
-		await this._validatorsAPI.setGeneratorList(apiContext, initDelegates);
+		const bftValidators = [];
```

Comment:
The code regarding setting BFT parameters and generators here looks good. There is one difference though: before we only were setting new BFT parameters if actually the set of active validators changed. This will have the effect that a certificate has to be generated for every round and we do not skip over rounds without a change in active validators. This is a minor disadvantage though and I find it acceptable.

Reply:
Here are the relevant lines in the certificate validation:
```
    heightNextBFTParameters = bftModule.getNextHeightBFTParameters(bftModule.getBFTHeights().maxHeightCertified+1)
    if previous function call returns valid height and aggregateCommit.height > heightNextBFTParameters-1:
        return False
```
see also https://github.com/LiskHQ/lips/blob/main/proposals/lip-0061.md#block-verification

Reply:
@janhack 
Updated: https://github.com/LiskHQ/lisk-sdk/pull/7169/commits/ff3be7ccdb9a38172be5438c2d6fc4a7073c9d46

- Updated to keep the BFTParams to only holds BLS key, and only updated when there is change
  - Also, the validators here is sorted lexicographically as before
- Created new generatorKeys store which holds address and generator key

Reply:
Looks all good!